frontend/src/app/main/ui/inspect/attributes/common.scss

```diff
@@ -94,6 +98,7 @@
     text-align: left;
     height: $s-16;
     color: var(--menu-foreground-color-rest);
+    background-color: red;
```

Comment:
Is this static value correct?
I am not being able to see this CSS applied on this sidebar, is it working?

Reply:
Good catch, that code was not being used. I have checked, on asset color, gradiend an regular color too.